frontend/src/types/stable-coin.ts

```diff
@@ -18,16 +18,17 @@ export type Transfers = {
 
 // types.ts
 export enum TransactionFilterOption {
+	Top10 = 10,
 	Top20 = 20,
 	Top50 = 50,
 	Top100 = 100,
 }
 
 // Optional: If you want labels for UI
 export const transactionFilterOptions = [
```

Comment:
I suggest , lets not change top to last as transactions comes based on recent time stamps. Here top signifies , recent transactions .